integration-tests/spring-data-jpa/src/main/java/io/quarkus/it/spring/data/jpa/CarRepository.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.it.spring.data.jpa;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface CarRepository extends JpaRepository<Car, Integer> {
+
+    List<Car> findByBrand(String brand);
+
+    @Query("SELECT m.model FROM MotorCar m WHERE m.brand = :brand")
+    List<String> findModelsByBrand(@Param("brand") String brand);
+}
```

Comment:
@geoand Hi Georgios!
Do you mean something like this?

Reply:
I was thinking something like `findByBrand` without a `Query`. I want to make sure that the proper query will be generated

Reply:
@goand Sorry, I don't get it. `findByBrand` is a custom method and does not have a `@Query` annotation. If I remove the patch that method fails. What path do you want to have tested which is not covered? Do you want a built in repository method to be tested? (like `findById`) Or do you want a more complex query?

Reply:
I just want to make sure that a non interface method works in the same way as findById for example

Reply:
So I'll add `Car findByBrandAndModel(String brand, String model)`? And also test findById?

Reply:
Sounds great 👍

Reply:
Better?

Reply:
kk